src/components/MARCTable/MARCTableRow.js

```diff
@@ -283,11 +258,11 @@ export const MARCTableRow = ({
               style={findContainerStyle}
             >
               <Label style={labelStyle}>
-                <FormattedMessage id="ui-data-import.settings.mappingProfile.marcTable.data.find" />
+                {formatMessage({ id: 'ui-data-import.settings.mappingProfile.marcTable.data.find' })}
```

Comment:
Could you please clarify why you started using this approach? I would say that `formatMessage` approach should be used only when `<FormattedMessage />` is not suited.

Reply:
I used `formatMessage` for rendering such props as `placeholder`, option label and so on, so I decided just get rid of `FormattedMessage` import and use `formatMessage` instead in this component. There is no problem to return `FormattedMessage`. Do you think this approach is better?

Reply:
I remember there were a discussion about getting away from `formatMessage` approach. When I find a link I will provide it here. Only based on that I would recommend prefer using `FormattedMessage` whenever possible like in this case. Once it is not, then `formatMessage` is good to go.

Reply:
I got it, thank you!